src/components/rocm/README.md

```diff
@@ -85,6 +85,20 @@ setting the ROCP\_TOOL\_LIB to the PAPI library as follows:
 
     The binary image of a `double` is intact; but users must recast to `double` for display purposes.
 
+* The following ROCm events are known to cause an error when the rocm component is used in sampling mode
+
+    TA_BUSY_{sum,avr,min,max}
```

Comment:
A similar memory access fault occurs with the event "TCP_TCC_NC_ATOMIC_REQ_sum." Perhaps re-word the README to not specify just the TA_BUSY_* events.

Reply:
I changed the wording. Can you also create an issue that we can reference in the PR?

Reply:
https://github.com/icl-utk-edu/papi/issues/115